README.md

```diff
@@ -47,16 +44,25 @@ folding, traffic flow optimization, job-shop scheduling, and many more.
 
 ## Installation
 
-You can run this example
-[in the Leap IDE](https://ide.dwavesys.io/#https://github.com/dwave-examples/structural-imbalance-notebook).
+You can run this example without installation in cloud-based IDEs that support 
+the [Development Containers specification](https://containers.dev/supporting)
```

Comment:
Since you're explicitly calling out `ocean-devcontainer` and Codespaces, should we mention the new auth, or link to the new IDE docs?

Reply:
This is a very interesting comment. In all our https://github.com/dwave-examples examples we assume the user has set up a SAPI token (in conf file or env var) except if the LEAP IDE is used, but it is possible that someone would try an example without setting up for Leap access. I added a paragraph under Usage, take a look.